.github/actions/docs-spelling/allow/keywords.txt

```diff
@@ -50,9 +52,12 @@ Fortuneo
 gebabebb
 GEBABEBB
 Greenshot
+GTQ
 HSA
 htpasswd
 IBANs
+icns
```

Comment:
Although this is technically correct I wonder if putting this word into the spelling file would make typos easier down the road. Maybe we can reword that PR description to avoid mentioning .icns?